tests/functional/unit_testing/test_unit_testing.py

```diff
@@ -118,5 +124,52 @@ def test_basic(self, project):
         results = run_dbt(["run"])
```

Comment:
^ can't comment one line above but - I don't think the `run_dbt(["deps"])` is necessary here